bundle/config/bundle.go

```diff
@@ -28,4 +28,12 @@ type Bundle struct {
 	// Contains Git information like current commit, current branch and
 	// origin url. Automatically loaded by reading .git directory if not specified
 	Git Git `json:"git,omitempty"`
+
+	// Determines the mode of the environment.
+	// For example, 'mode: development' can be used for deployments for
+	// development purposes.
```

Comment:
Please add a short line why this is annotated as read only. It took me a second to realise this is because it must be set at the environment level, not the bundle level.